src/canopy-react-error-boundary.js

```diff
@@ -62,6 +62,16 @@ export default function decorateOptions(opts) {
         }
       }
       componentDidCatch(err, info) {
+        if (info && info.componentStack) {
+          err.extra =  Object.assign(err.extra || {}, {
```

Comment:
Adding an `extra` property sends extra data to sentry: https://code.canopy.dev/front-end/infrastructure/sentry-error-logging-ui/blob/88cbd5e9a3b3834a20324c23320c33cd4323f665/src/sentry.helper.js#L90